src/main/java/wolox/training/utils/ErrorConstants.java

```diff
@@ -0,0 +1,32 @@
+package wolox.training.utils;
+
+public class ErrorConstants {
+
+  //PRECONDITIONS BOOK
+  public static String NOT_NULL_GENRE = "The genre field cannot be null.";
+  public static String NOT_EMPTY_GENRE = "The genre field cannot be empty.";
```

Comment:
You can implement a generic string here. Something like this :
`The %s field cannot be empty`
And then do:
`String.format(ErrorConstants.fieldEmpty, "genre")`